src/healthRecord/healthRecord.interface.ts

```diff
@@ -46,6 +47,8 @@ export class AddHealthrecord {
   @IsString()
   hrName: string
   @ApiProperty({ type: UploadImage })
+  @IsOptional()
+  @IsObject()
   picture?: UploadImage
   @ApiProperty({ type: [HealthDataField] })
   @IsArray()
```

Comment:
Just curious why there are the same class in both dto and interface. Is there any reason?

Reply:
I wanted to separate dto for controller and interface for service